cmd/metrics-server/app/options/options_test.go

```diff
@@ -0,0 +1,138 @@
+package options
```

Comment:
we might want to add a license header here

Reply:
Nice catch, It would be great to automate license validation

Reply:
Filed https://github.com/kubernetes-sigs/metrics-server/issues/596